pkgconfig/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if(NOT DISABLE_PACKAGE_CONFIG_AND_INSTALL)
+  cmake_path(APPEND CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_INCLUDEDIR}" OUTPUT_VARIABLE PKGCONFIG_INCLUDEDIR)
+  cmake_path(APPEND CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_LIBDIR}" OUTPUT_VARIABLE PKGCONFIG_LIBDIR)
```

Comment:
`cmake_path(APPEND)` requires CMake 3.20.0, I think it'd be more appropriate for mbedtls to just include the [JoinPaths.cmake](https://github.com/jtojnar/cmake-snips/blob/master/CMakeScripts/JoinPaths.cmake) script. It's very permissively licensed (public domain / MIT)

Reply:
yes that's why I had this is draft to get a comment if they wanted to bump their min version to 3.20  (which is March 2021 release and almost 3 years old) or pull in that script. But since it's a licensing issue, it's their decision. IMO just bump to 3.20.